pkg/theia/commands/anomaly_detection_algos.go

```diff
@@ -0,0 +1,226 @@
+// Copyright 2022 Antrea Authors
```

Comment:
I just kept it as also as it is about running the specific algorithm rather than running the command without algo

Reply:
I agree with Yanjun here, `anomaly_detection_run.go` is a better name because it can be consistent with our command.
Regarding to your concern, we can set a default algorithm like ARIMA for this command.